### ISSUE TITLE
empty output

### ISSUE THREAD

**Issue by hoseinsalehi** [KMarkert/servir-vic-training#9]

**Comment by hoseinsalehi** [comment]
hello sir
I started implementing a Python code model, but a few outputs were zero and no routing was done.
Rainfall and runoff outputs and water balance are zero.
I have attached the output file.
[outputs.zip](https://github.com/KMarkert/servir-vic-training/files/2683765/outputs.zip)

**Comment by KMarkert** [comment]
Hi Hosein,
I took a look at the outputs you provided. It seems that the runoff and precip files have results in the data arrays. Your snow water equivalent results will be 0 in the Nyando region because it is tropical.
Can you please provide the command and files that you are using for the routing model? I can check to see if the data and command are correct for you.

**Comment by hoseinsalehi** [comment]
hello sir
I appreciate your attention.
I placed the input files for routing in the following file.
I used the following instructions exactly as the text in the guide you provided.
I posted a screenshot of the error I encountered.
I am waiting for your response.
[rout inputs.zip](https://github.com/KMarkert/servir-vic-training/files/2735779/rout.inputs.zip)
![rout](https://user-images.githubusercontent.com/38522226/50818729-266b2680-133d-11e9-9fff-f6427d90a56c.png)
